pytests/repo/tdnf-test-cleanreq-required.spec

```diff
@@ -0,0 +1,34 @@
+#
+# tdnf-test-one spec file
+#
+Summary:    basic install test file.
+Name:       tdnf-test-cleanreq-required
+Version:    1.0.1
+Release:    3
+Vendor:     VMware, Inc.
+Distribution:   Photon
+License:    VMware
+Url:        http://www.vmware.com
+Group:      Applications/tdnftest
+
+%description
+Part of tdnf test spec. Basic install/remove/upgrade test
+
+%prep
+
+%build
+
+%install
+mkdir -p %_topdir/%buildroot/lib/cleanreq/
+touch %_topdir/%buildroot/lib/cleanreq/required
+
+%files
+/lib/cleanreq/required
+
+%changelog
+*   Thu Nov 04 2021 Oliver Kurth <okurth@vmware.com> 1.0.1-3
```

Comment:
use 1.0.1-1 no need of other changelogs???

Reply:
> -> Two installed packages X & Y
-> Both require Z
-> Removing X should not remove Z
-> Removing y should remove Z
-> Removing both X & Y together should remove Z

That's what `test_asutoremove2` does.

Reply:
It's just a test file. It doesn't really matter.